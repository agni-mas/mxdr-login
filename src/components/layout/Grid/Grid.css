.grid {
    display: grid;
    grid-template-columns: 2fr 1fr;
    gap: 19px;
    margin: 31px 0;
    position: relative;
}