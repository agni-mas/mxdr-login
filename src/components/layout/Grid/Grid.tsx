import React, { FC } from 'react';
import './Grid.css';

interface GridProps {
  children: any
}

const Grid: FC<GridProps> = ({ children }) => (
  <div className='grid'>
    {children}
  </div>
);

export default Grid;
