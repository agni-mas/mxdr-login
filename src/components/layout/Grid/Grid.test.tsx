import { render } from "@testing-library/react";
import Grid from "./Grid";

it("renders <Grid/>", () => {
    const props = {
        children: <div>Children</div>
    }
    const { asFragment } = render(<Grid children={props.children} />);
    expect(asFragment()).toMatchSnapshot();
});