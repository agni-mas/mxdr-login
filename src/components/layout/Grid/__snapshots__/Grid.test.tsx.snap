// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <Grid/> 1`] = `
<DocumentFragment>
  <div
    class="grid"
  >
    <div>
      Children
    </div>
  </div>
</DocumentFragment>
`;
